packages/tocco-ui/src/ButtonGroup/ButtonGroup.js

```diff
@@ -12,7 +12,8 @@ const ButtonGroup = props => {
     <StyledButtonGroup melt={props.melt}>
       { // eslint-disable-next-line
         React.Children.map(props.children, child => {
-          return React.cloneElement(child, {look: props.look, buttonGroupInk: props.ink, buttonGroupMelt: props.melt})
+          return child !== null
```

Comment:
I personally wouldn't use type safe null checks in cases like this. The code becomes more stable if you remove `!== null`.